splice_machine/src/main/java/com/splicemachine/derby/utils/SpliceAdmin.java

```diff
@@ -2276,9 +2274,29 @@ public static void INVALIDATE_GLOBAL_DICTIONARY_CACHE() throws Exception {
     public static void SHOW_CREATE_TABLE(String schemaName, String tableName, ResultSet[] resultSet) throws SQLException
     {
         List<String> ddls = SHOW_CREATE_TABLE_CORE(schemaName, tableName, false, true);
-        StringBuilder sb = new StringBuilder("SELECT * FROM (VALUES '");
-        sb.append(ddls.get(0));
-        sb.append(";') FOO (DDL)");
+        StringBuilder sb = new StringBuilder("SELECT * FROM (VALUES ");
```

Comment:
You should (over)estimate the capacity of the `StringBuilder` otherwise you leave it grow exponentially which could stress the GC e.g. if you have hundreds of such queries running on parallel.

i.e. do something like this:
```
long size = 22 /* size of 'SELECT * FROM (VALUES ' */ + 11 /* size of ') FOO (DDL)' */ + ddl.length + 15 /* size of 'cast('' as clob)'*/*(ddl.length / MAX_STR_LITERAL_LENGTH) + 7 /* size of '|| ';''*/
StringBuilder sb = new StringBuildre(size);
```